pkg/cmd/server/api/helpers.go

```diff
@@ -398,7 +398,7 @@ func DefaultClientTransport(rt http.RoundTripper) http.RoundTripper {
 		Timeout:   30 * time.Second,
 		KeepAlive: 30 * time.Second,
 	}
-	transport.Dial = dialer.Dial
```

Comment:
I don't know exactly what this does.  Remove this particular change and open an issue to figure out why we have our own DefaultClientTransport.